docs/content/howto/notebook.md

```diff
@@ -4,58 +4,132 @@ order: 600
 description: How to embed Rerun in notebooks like Jupyter or Colab
 ---
 
-Starting with version 0.5.0, Rerun now has limited support for embedding the Rerun viewer directly within IPython-style
+Starting with version 0.15.1, Rerun has improved support for embedding the Rerun viewer directly within IPython-style
 notebooks. This makes it easy to iterate on API calls as well as to share data with others.
 
 Rerun has been tested with:
- - [Jupyter Notebook Classic](https://jupyter.org/)
- - [Jupyter Lab](https://jupyter.org/)
- - [VSCode](https://code.visualstudio.com/blogs/2021/08/05/notebooks)
- - [Google Colab](https://colab.research.google.com/)
+
+-   [Jupyter Notebook Classic](https://jupyter.org/)
+-   [Jupyter Lab](https://jupyter.org/)
+-   [VSCode](https://code.visualstudio.com/blogs/2021/08/05/notebooks)
+-   [Google Colab](https://colab.research.google.com/)
 
 ## Basic concept
 
-Rather than logging to a file or a remote server, you can also configure the Rerun SDK to store data in a local
-[MemoryRecording](https://ref.rerun.io/docs/python/stable/common/other_classes_and_functions/#rerun.MemoryRecording).
+When using the Rerun logging APIs, the output is buffered in-memory until you send it to a sink.
+There is a helper method: [`rr.notebook_show()`](https://ref.rerun.io/docs/python/stable/common/initialization_functions/#rerun.notebook_show), which takes any buffered messages, converts
+them into an embedded rrd file and emits an HTML snippet using the hosted rerun viewer that
+references the embedded data.
```

Comment:
that's useful information but it's hard to understand how that relates to the sink bit